torch/signal/windows/windows.py

```diff
@@ -759,3 +761,64 @@ def general_hamming(M,
                           layout=layout,
                           device=device,
                           requires_grad=requires_grad)
+
+
+@_add_docstr(
+    r"""
+Computes the minimum 4-term Blackman-Harris window according to Nuttall.
```

Comment:
Formula?

Reply:
Sure, it's a sum of cosines. I'll add it.